permissions/plugin-wavefront.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "wavefront"
+github: "jenkinsci/wavefront-plugin"
+paths:
+- "com/vmware/devops/plugins/wavefront"
```

Comment:
This path doesn't match the `groupId` / `artifactId` from your pom.xml

https://github.com/jenkinsci/wavefront-plugin/blob/master/pom.xml#L14

`plugins` doesn't appear in either the groupId or artifactId.

Reply:
Okay, I will fix that